etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/localfile/LocalEndpointClient.java

```diff
@@ -0,0 +1,51 @@
+package gov.hhs.cdc.trustedintermediary.external.localfile;
+
+import gov.hhs.cdc.trustedintermediary.etor.RSEndpointClient;
+import gov.hhs.cdc.trustedintermediary.external.reportstream.ReportStreamEndpointClientException;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import javax.annotation.Nonnull;
+
+public class LocalEndpointClient implements RSEndpointClient {
```

Comment:
Hmmm, I'm thinking of renaming this to `MockRsEndpointClient`.  I don't want this class to be confused for a locally running (in Docker or whatever) version of ReportStream.

Reply:
I second that idea.

Reply:
I'm ok with that. I was trying to follow the naming convention used for `LocalFileOrderSender`, but I agree `MockRsEndpointClient` it's clearer. I'll change it

Reply:
Renamed